packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -192,8 +193,9 @@ export const InputBoxButton = (props: InputBoxButtonProps): JSX.Element => {
         onMouseLeave={() => {
           setIsHover(false);
         }}
-        onRenderIcon={() => onRenderIcon(isHover)}
       />
```

Comment:
Maybe update this to a div since it isn't behaving like a IconButton (possibly try a default button as well)

Reply:
Thanks. Went with DefaultButton because it has the ariaLabel prop.